src/app/questions/file/file-display/file-display.component.ts

```diff
@@ -25,6 +27,7 @@ export class FileDisplayComponent implements OnInit, OnChanges {
   private resetUILogic() {
     this.virusDetection = {};
     this.quarantine = {};
+    this.video?.nativeElement.load();
```

Comment:
is the question mark intended?

Reply:
yes, if I remove that I'll get this error:
ERROR TypeError: Cannot read property 'nativeElement' of undefined